apps/tai/lib/tai/advisor.ex

```diff
@@ -74,177 +71,67 @@ defmodule Tai.Advisor do
 
       def init(state), do: {:ok, state, {:continue, :started}}
 
+      def handle_info({:tai, %Quote{} = event}, state) do
+        key = {event.venue_id, event.product_symbol}
+        new_data = Map.put(state.market_quotes.data, key, event)
+        new_market_quotes = Map.put(state.market_quotes, :data, new_data)
+        new_state = Map.put(state, :market_quotes, new_market_quotes)
+
+        {
+          :noreply,
+          new_state,
+          {:continue, {:execute_event, event}}
+        }
+      end
+
       def handle_continue(:started, state) do
         after_start(state)
 
         state.products
-        |> Enum.each(fn p ->
-          Tai.PubSub.subscribe([
-            {:order_book_snapshot, p.venue_id, p.symbol},
-            {:order_book_changes, p.venue_id, p.symbol}
-          ])
-        end)
+        |> Enum.each(&Tai.PubSub.subscribe({:market_quote, &1.venue_id, &1.symbol}))
```

Comment:
I think subscription to `market_quote` can be be made optional using config

Reply:
👍 I was intending to make this optional in a separate PR when I formalize the subscription boot steps. e.g. with a `subscribe` callback.